osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -623,17 +620,11 @@ protected override void UpdateAfterChildren()
         protected internal new ScheduledDelegate Schedule(Action action) => base.Schedule(action);
 
         /// <summary>
-        /// A safe offset prior to the start time of <see cref="HitObject"/> at which this <see cref="DrawableHitObject"/> may begin displaying contents.
+        /// A offset prior to the start time of <see cref="HitObject"/> at which this <see cref="DrawableHitObject"/> may begin displaying contents.
```

Comment:
it should be 'An' instead of 'A' now since the word 'safe' was removed and the next word starts with a vowel.